src/js/ui/window.js

```diff
@@ -279,9 +280,7 @@ Window.prototype._emit = function(type, subtype, e) {
   if (klass) {
     e[klass._codeName] = this;
   }
-  if (this.emit(type, subtype, e) === false) {
-    return false;
-  }
+  return !!this.emit(type, subtype, e);
```

Comment:
this change allows the browser to know if a handler acted on the event - in the case of the back button we want to know if there was no handler so we can take action